client/my-sites/checkout/src/components/wp-order-review-line-items.tsx

```diff
@@ -392,6 +398,14 @@ function LineItemWrapper( {
 			return true;
 		}
 
+		if (
+			isWpComPlan( variant.productSlug ) &&
+			! isStreamlinedPriceExperimentLoading &&
+			isStreamlinedPriceRadioTreatment( streamlinedPriceExperimentAssignment )
+		) {
+			return true;
```

Comment:
We might want to note this in the PR description, since this changes a behavior that was specifically requested by the Finance team. I think it's probably fine to do, personally, but it is potentially a big change for checkout and may drive more monthly plan purchases which was something that Finance thought we should be discouraging. It may affect your experiment results more than you think.